decidim-admin/spec/jobs/export_job_spec.rb

```diff
@@ -2,38 +2,37 @@
 require "spec_helper"
 
 module Decidim
-  module Proposals
+  module Admin
     describe ExportJob do
```

Comment:
Maybe not for this PR but I think we could create a ExportJob examples so every feature can include them in its test suite like we did for comments.